cmd/pow/cmd/data_summary.go

```diff
@@ -0,0 +1,43 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"google.golang.org/protobuf/encoding/protojson"
+)
+
+func init() {
+	dataCmd.AddCommand(dataSummaryCmd)
+}
+
+var dataSummaryCmd = &cobra.Command{
+	Use:   "summary [optional cid1,cid2,...]",
+	Short: "Get a summary about the current storage and jobs state of cids",
+	Long:  `Get a summary about the current storage and jobs state of cids`,
+	Args:  cobra.MaximumNArgs(1),
```

Comment:
Couldn't this CLI receive cid1, cid2, ..., cidN. 
`MaximumNArgs(1)` is the right setup?

Reply:
Yea technically it's one string arg that contains commas, and we split the arg on the commas.